src/hyp3_isce2/__main__.py

```diff
@@ -14,15 +17,19 @@ def main():
     parser = argparse.ArgumentParser(prefix_chars='+', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument(
         '++process',
-        choices=[
-            'insar_tops_burst',
-            'insar_stripmap'
-        ],
+        choices=['insar_tops_burst', 'insar_stripmap'],
         default='insar_tops_burst',
         help='Select the HyP3 entrypoint to use',  # HyP3 entrypoints are specified in `pyproject.toml`
     )
 
     args, unknowns = parser.parse_known_args()
+
+    # NOTE: Earthdata username and password should be provided as environment variables, or be present in your .netrc
```

Comment:
This comment should probably be moved to the README for better visibility to users.

Reply:
Or perhaps it's not needed at all, if users always pass these to the Docker container?

Reply:
I added a note to the readme, since it's worth covering this up-front.